### DISCUSSION TITLE
Uncaught ReferenceError: Livewire is not defined and ERR_HTTP2_PROTOCOL_ERROR 200

### DISCUSSION THREAD

**Discussion by jaguar988** [livewire/livewire#4693] [1. Help]
i have followed all the threads regarding this issue but i  did not solve it only on ovh servers
after setting laravel i published the assets with php artisan livewire:publish --assets i modified the livewire.php file  'asset_url'=>env('App_URL', 'https://mydomain.com/ss'), also i added the livewire tags to blade file @livewireStyles and @livewireScripts

and integrated livewire in blade as following: 
@livewire('create-reglements', ['pid'=> $pid])

 
when i check source code where livewire exists i found the livewire script is correctly loaded
<script src="https://mydomain/ss/livewire/livewire.js?id=9a36ebbddb8dd0aa91b1" data-turbo-eval="false" data-turbolinks-eval="false" ></script>

when i check the console it shows these errors:
ajouter:411          GET https://mydomain/mz/livewire/livewire.js?id=9a36ebbddb8dd0aa91b1 net::ERR_HTTP2_PROTOCOL_ERROR 200
and
Uncaught ReferenceError: Livewire is not defined after checking the source of that issue i found that  Livewire Class is not recognized window.livewire = new Livewire();

UPDATE: I just found that ssl cause this issue, i added protected $proxies= '*'; in middleware; but still not working
any suggestion is much appreciated thank you!

**Comment by joshhanley** [comment]
@jaguar988 have you tried it without publishing the javascript assets? I would not recommend publishing the assets unless you really have to (using a CDN or something).

**Reply by jaguar988** [reply]
Yes i tried without publishing the js assets and it does not work. this issue only happened in ovh server i have tried before digitalocean it worked for me

**Reply by jaguar988** [reply]
> @jaguar988 have you tried it without publishing the javascript assets? I would not recommend publishing the assets unless you really have to (using a CDN or something).

@joshhanley  just found the issue its ssl that cause it i remove it from .env and it works but how can i use ssl with livewire?

**Reply by joshhanley** [reply]
@jaguar988 ah interesting. I've never had an issue with using SSL with Livewire. Can you explain in more detail what you meant by you removed it from the `.env`?
